### PR TITLE
ChangeRecord: rename category into protocol

### PR DESCRIPTION
**Pull request by jecisc**
Rename #category into #protocol in ChangeRecord

### CODE REVIEW

File: src/Monticello/ChangeRecord.extension.st [hunk 1]
```diff
@@ -10,7 +10,7 @@ ChangeRecord >> asMCMethodDefinition [
 		className: class
 		classIsMeta: meta
 		selector: self methodSelector
-		category: category
+		category: protocol
```
Comment (privat) [inline src/Monticello/ChangeRecord.extension.st:13]: the keyword is still `category:` -_-

File: src/System-Changes/ChangeRecord.class.st [hunk 2]
```diff
@@ -69,26 +64,18 @@ ChangeRecord >> file: f position: p type: t class: c category: cat meta: m stamp
...
+		self methodClass ifNotNil: [ :methodClass |
+			methodClass
+				compile: self string
+				classified: protocol
+				withStamp: stamp
+				notifying: nil ].
+		type == #doIt ifTrue: [
+			| string |
+			string := self string.
+			(string beginsWith: '----') ifFalse: [ self class compiler evaluate: string ] ].
```
Comment (privat) [inline src/System-Changes/ChangeRecord.class.st:77]: `'----'` I'm afraid to ask, but what the heck?

**Review by privat — APPROVED** [review]

File: src/Monticello/ChangeRecord.extension.st (hunk 1 above)
Comment (jecisc) [reply src/Monticello/ChangeRecord.extension.st:13]: Yes but MCMethodDefinition is next on the list ahah

I'm trying to do small PRs to help with review and not take too much risk of breaking everything.

File: src/System-Changes/ChangeRecord.class.st (hunk 2 above)
Comment (jecisc) [reply src/System-Changes/ChangeRecord.class.st:77]: No idea... I didn't really want to know ahah

**Review by MarcusDenker — APPROVED** [review]

**Pull request merged**